ex1/functions.py

```diff
@@ -90,26 +127,62 @@ def f():
 # Define a function named args_to_dict that takes three arguments and returns a
 # dictionary with the position of the argument as the key (starting at 0) and
 # the argument as the value.
-
+def args_to_dict(x,y,z):
+    mydict = {0 : x, 1 : y, 2 : z}
+    return mydict
 # BONUS: Write a function named args_to_dict_general that does the same for any
 # number of arguments
-
+def args_to_dict_general(*x):
+    mydict = {}
+    i = 0
+    for y in x:
+        mydict[i] = y
+        i = i + 1
```

Comment:
You can use `enumerate` here then there is no risk of forgetting the manual increment of `i`